### PR TITLE
14.0 [MIG] account_invoice_pricelist_sale: Migration to 14.0

### PR DESCRIPTION
**Pull request by hildickethan**
Standard migration to 14.0

- [x] depends on https://github.com/OCA/account-invoicing/pull/850

### CODE REVIEW

File: account_invoice_pricelist_sale/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def _prepare_invoice(self):
+        """Pricelist_id is set on invoice."""
+        self.ensure_one()
+        val = super(SaleOrder, self)._prepare_invoice()
```
Comment (alexis-via) [inline account_invoice_pricelist_sale/models/sale_order.py]: you can write super()._prepare_invoice()

**Comment by hildickethan** [comment]
rebased

File: account_invoice_pricelist_sale/models/sale_order.py (hunk 1 above)
Comment (hildickethan) [reply account_invoice_pricelist_sale/models/sale_order.py]: done

**Review by florian-dacosta — APPROVED** [review]

**Review by Abranes — APPROVED** [review]

**Comment by hildickethan** [comment]
@dreispt I rebased it, would the merge succeed now? Not sure what to do with the coverage since the module doesn't do much and it should be covered by the test that already exists

**Pull request merged**